scripts/build/builders/bouffalolab.py

```diff
@@ -107,6 +107,12 @@ def __init__(self,
         elif enable_shell:
             self.argsOpt.append('chip_build_libshell=true')
 
+        try:
+            self.argsOpt.append('bouffalolab_sdk_root="%s"' % os.environ['BOUFFALOLAB_SDK_ROOT'])
+        except KeyError:
+            # environment BOUFFALOLAB_SDK_ROOT is not existed, try default setup path
+            self.argsOpt.append('bouffalolab_sdk_root="/opt/bouffalolab_sdk"')
```

Comment:
Do not set such defaults in scripts/build/builders

Reply:
At a minimum you should check if `/opt/bouffalolab_sdk` exists and if yes, then you can set it. Otherwise you should not.

If you just set it and it does not exist, the resulting build error changes are likely hard to debug since they will be somewhere inside GN.

Reply:
Trying on default path is removed.
For developer, export BOUFFALOLAB_SDK_ROOT=/opt/bouffalolab_sdk is required before building executed, which is mentioned in example Readme.md.